http/mediatypes/mediatypes.go

```diff
@@ -0,0 +1,7 @@
+package mediatypes
+
+// Media types
+const (
+	ApplicationJSON     = "application/json"
```

Comment:
Since this shouldn't be exported into the public API and we don't anticipate a whole lot of "application" media types nevertheless JSON ones, just drop "Application" from these names, leaving: "JSON" and "JSONUTF8" (see below).

Reply:
outdated: I've removed `mediatypes` package